Simplenote/PDFLinkActivityItemProvider.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public class PDFLinkActivityItemProvider : UIActivityItemProvider {
+    public let targetURL: URL
+    public let filename: String
+    public let note: Note
+    
+    public init(note: Note) {
+        self.note = note
+        self.filename = String(format: "%@.pdf", "Thisisastring")
+        self.targetURL = FileManager.documentsURL.appendingPathComponent(filename)
```

Comment:
I was thinking that **PERHAPS** we got the WP exporter wrong. Can we switch to [temporaryDirectory](https://www.google.com/search?client=safari&rls=en&q=NSFileManager+temporaryDirectory&ie=UTF-8&oe=UTF-8) ? 

[Ref. here](https://stackoverflow.com/questions/16666812/whats-the-difference-between-saving-content-at-document-directory-or-temp-folder). Files stored in `temporary` aren't meant to persist thru launch OPs

Reply:
Second note: the `Target URL` is also calculated by `SimplenotePDFExporter`. It'd be neat to converge everything in a single spot 🔥